.github/workflows/wash-plugins.yml

```diff
@@ -7,25 +7,20 @@ on:
 
 permissions:
   contents: read
+  packages: write
 
 jobs:
-
   build:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@d632683dd7b4114ad314bca15554477dd762a938
-    - name: Extract tag context
-      id: ctx
-      run: |
-          version=${GITHUB_REF_NAME#wash-plugin-v}
-          echo "version=${version}" >> "$GITHUB_OUTPUT"
-          echo "version is ${version}"
+    - uses: ./.github/actions/configure-wkg
+        
     - name: Build
+      working-directory: crates/wash-lib
+      run: |
+        wkg wit build -o package.wasm
+    - name: Push version-tagged WebAssembly binary to GHCR
+      working-directory: crates/wash-lib
       run: |
-        cd crates/
-        tar czf wash-plugins-${{ steps.ctx.outputs.version }}.tar.gz wash-lib/wit
-    - name: Release
-      uses: softprops/action-gh-release@c062e08bd532815e2082a85e87e3ef29c3e6d191
-      with:
-        files: crates/wash-plugins-${{ steps.ctx.outputs.version }}.tar.gz
-        make_latest: "false"
```

Comment:
What do you think about publishing to *both* places? I think the tar files are easier entry points for beginners, and a nice artifact to have. 

Also, in the meantime before absolutely everything switches to using `wkg`, we can update/do releases.

Reply:
I can but I think that is overhead we didn’t need anymore. We were the only ones really using wit-deps, and I think it is much easier to go to a repo to look at the wit than pull down a tarball. I don’t think any other major updates will be happening to these ones were publishing before we’ve moved most people to the newer version of wash. Is there another use case here I am missing? (Distinct possibility given how crazy the last few weeks have been)

Reply:
> I can but I think that is overhead we didn’t need anymore.

Hmnn that's interesting, I didn't really think about it as overhead -- I think of it as part of releasing the package. People do still package source code/RPMs/etc, even if a container image is available right?

> We were the only ones really using wit-deps, and I think it is much easier to go to a repo to look at the wit than pull down a tarball.

Ah I was thinking less about `wit-deps` and more about making the releases *have artifacts*. For example let's take the [latest WIT release for `bus`](https://github.com/wasmCloud/wasmCloud/releases/tag/wit-wasmcloud-bus-v1.0.0) -- there's *something* to pull down there. I'd like to maintain that, so maybe there's some other ways:

(1) Add a URL to where the textual WIT files are actually present (I guess in the release text?)
(2) Add the produced container image to the release 

This kind of gets to the issue I think -- if we did (2) I think people might be confused with what to do with the container and what to do with the files inside. It requires knowing a lot about the ecosystem/what to do, rather than just... "oh it's a bunch of files".

Also, I just realized but I made this comment under the WIT for `wash` plugins... The same change was made to many but I want to clarify that I am more focused on the other ones like postgres, secrets, and bus. Wash plugins have a different onboarding flow i think.

Reply:
I can add this back in then